src/components/ProfilePage/Common/IconHeader.tsx

```diff
@@ -9,16 +10,19 @@ export interface IconHeaderProps {
   tooltipKey: string;
   title: string;
   icon?: React.ReactNode;
+  sx?: SxProps<Theme>;
 }
 
-const IconHeader = ({ tooltipKey, title, icon }: IconHeaderProps) => {
+const IconHeader = ({ tooltipKey, title, icon, sx }: IconHeaderProps) => {
   const { t } = useTranslation();
   const theme = useTheme();
 
   return (
-    <IconHeaderContainer>
+    <IconHeaderContainer sx={sx}>
```

Comment:
Not really fan of passing sx through multiple custom components, can't we avoid it?